R/extract.R

```diff
@@ -197,6 +242,27 @@ extract.averaging <- function(model, use.ci = FALSE, adjusted.se = FALSE,
   return(tr)
 }
 
+#' \code{\link{extract}} method for \code{averaging} objects
+#'
+#' \code{\link{extract}} method for \code{averaging} objects. These objects are
+#' created by the \code{\link[MuMIn]{averaging}} function in the \pkg{MuMIn} package.
+#'
+#' @param model A model object.
+#' @param use.ci Report confidence intervals in the GOF block?
+#' @param adjusted.se Report adjusted standard error in the GOF block?
+#' @param include.nobs Report the number of observations in the GOF block?
+#' @param ... Additional arguments for the \code{\link[base]{summary}} function
+#'   for \code{averaging} objects.
+#' @inheritParams extract
```

Comment:
Inherit `model`, `include.nobs`, and `...` from `extract,lm-method` and remove line 256?